README.md

```diff
@@ -370,12 +370,12 @@ client
 
 ##### Create a Signed Search Key
 
-Creating a search key that will only return the title field.
+Creating a signed search key that will only return the title field. Note: choose a private key with read access enabled to sign the Signed Search Key.
```

Comment:
```suggestion
Signed search keys keep your read-only Private API Keys secret and restrict what a user can search over. They can be used to restrict top-level search options. The example below creates a signed search key that will only return the `title` field.

See https://www.elastic.co/guide/en/app-search/current/authentication.html#authentication-signed for more options and documentation.
```

Reply:
^ updated the copy to match the Elastic docs a bit more closely (helping note what signed search keys do) as well as links to the docs for more options